### DISCUSSION TITLE
[Question][Hw3] More example codes for Ocaml??

### DISCUSSION THREAD

**Discussion by Quasar0311** [prosyslab-classroom/cs348-information-security#529] [Q&A]
Can we get some more example codes for Ocaml (actually LLVM-used example might be better)?

Actually, I've seen several pages and that this assignment mentioned, but I couldn't understand some syntax of Ocaml and usage of LLVM libraries. Therefore, any more examples would be appreciated.

Thank you.

**Comment by KihongHeo** [comment]
For OCaml, there are already many examples in the [tutorial](https://www.cs.cornell.edu/courses/cs3110/2019sp/textbook/) which I put in the README,
including more advanced examples such as [stack](https://www.cs.cornell.edu/courses/cs3110/2019sp/textbook/modules/ex_stacks.html).

Also please take a look at the example in the LLVM primer. Based on that, try slightly different things by yourself.

Were you able to complete the warming up HW? 

If you still need more help, please use office hours.
